client/src/pages/Login.jsx

```diff
@@ -1,9 +1,118 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  Form, Input, Button,
+  message,
+} from 'antd';
+import Img from '../components/common/Img';
+// import Title from '../components/common/Title';
+import flower from './img/flower.png';
+import './login.css';
+import http from '../services/httpService';
 
-function Login() {
+const { Item } = Form;
+const Login = () => {
+  const [value, setValue] = useState({ email: '', password: '' });
+  const history = useHistory();
+  const sendRequest = async () => {
+    try {
+      if (value.email && value.password.length > 6) {
+        const response = await http.post('api/v1/login', value);
+        if (response) {
+          message.success('تم تسجيل الدخول بنجاح');
+          history.push('/');
```

Comment:
Good Job Nadia,
we do need to setUser to user from the authProvider, so maybe we'll need to send user data from the server so we can setUser({id, name, role})